src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,22 @@
+package duke;
+
+/**
+ * Parent Exception to all custom exceptions for the Duke project.
+ */
+public class DukeException extends Exception {
+    /**
+     * Constructor for a DukeException.
+     */
+    public DukeException() {
+        super();
+    }
+
+    /**
+     * Constructor for a DukeException with an optional message.
+     *
+     * @param message Exception message.
+     */
+    public DukeException(String message) {
+        super(message);
+    }
+}
```

Comment:
I feel like you can put all the exceptions(DukeExeption, EmptyBodyException, InvalidIndexException) under a package called DukeMainExceptions.